src/common/ui-wrapper.js

```diff
@@ -16,7 +16,6 @@ class UIWrapper {
 
   constructor(player: KalturaPlayer, options: KPOptionsObject) {
     const config: KPUIOptionsObject = options.ui;
-    config.uiComponents = [...(player.uiComponents || []), ...(config.uiComponents || [])];
```

Comment:
it is possible for app to add components as well so can't remove this

Reply:
why not? it just merge the plugins uiComponents with config uiComponents, it shouldn't change and then uiComponents will pass as is

Reply:
need to make sure that if app passes the uiComponents config it is passed.
We only discussed the plugins which is internal, but an app can do:
```javascript
KalturaPlayer.setup({
  plugins: {
    uiComponents: []
  }
})
```

Reply:
Yes, sure. Why not? Nobody remove it, it pass on config directly to ui manager as it was before.
Those lines merging the uiComponents from plugins and config.uiComponents which doesn't need it anymore its done by addComponent API instead

Reply:
```
KalturaPlayer.setup({
  ui: {
    uiComponents: []
  }
})
```
@OrenMe

Reply:
correct :-)

Reply:
so we're good, right?

Reply:
No, app provided config will stop working

Reply:
Got ya, you are correct, the config is still passing 👍

Reply:
So `player.uiComponents` is useless. maybe we can remove it (don't think anyone uses it)

Reply:
Didn't remove it since it's API

Reply:
I think we only used it when we passed the uiComponents from playkit when plugins were in it.
https://github.com/kaltura/kaltura-player-js/pull/264
It is safe to remove as we alway supported adding it only on initial config until we recently added the dynamic injection API which has it's own code paths.

Reply:
I mean if customers use it as getter, anyway if it's ok by you I'll remove it

Reply:
@esakal can you please review the removal of `uiComponents` getter.
It seems this is only used for setting it but maybe `contrib` is using it?